src/components/Tree/TreeItem/TreeItem.tsx

```diff
@@ -307,6 +306,10 @@ export const TreeItem = memo(
             </li>
         );
     },
+    (prevProps, nextProps) => {
+        console.log(prevProps, nextProps);
+        return false;
+    },
```

Comment:
do we need this?

Reply:
Oh no. Sorry. Will remove it. Good catch

Reply:
Done